Source/WebCore/dom/Document.cpp

```diff
@@ -3578,8 +3578,11 @@ const URL& Document::urlForBindings() const
         if (m_url.url().isEmpty() || !loader() || !isTopDocument() || !frame())
             return false;
 
-        auto* topDocumentLoader = topDocument().loader();
-        if (!topDocumentLoader || !topDocumentLoader->networkConnectionIntegrityPolicy().contains(NetworkConnectionIntegrity::Enabled))
+        auto* targetLoader = topDocument().loader();
```

Comment:
I'm not sure "target" a good word here.  "policyDocumentLoader" also has other meaning.  What about "policySource" or "policyContainer"?  Same elsewhere.

Reply:
Changed this to policySource.